content/bn/blue-green-deployment.md

```diff
@@ -0,0 +1,29 @@
+---
+title: নীল সবুজ স্থাপনা (Blue Green Deployment)
+status: Completed
+category: ধারণা
+tags: ["অ্যাপ্লিকেশন", "", ""]
+---
+
+## এটা কি
+
+ন্যূনতম ডাউনটাইম সহ চলমান কম্পিউটার সিস্টেম আপডেট করার জন্য নীল-সবুজ স্থাপনা একটি কৌশল।
+অপারেটর দুটি পরিবেশ বজায় রাখে, যা "নীল" এবং "সবুজ" নামে ডাকা হয়।
+একটি প্রোডাকশন ট্র্যাফিক পরিবেশন করে (বর্তমানে সমস্ত ব্যবহারকারী যে সংস্করণটি ব্যবহার করছেন), যখুন অন্যটি আপডেট করা হয়।
```

Comment:
Kindly create a  better sentence from this one : একটি প্রোডাকশন ট্র্যাফিক পরিবেশন করে (বর্তমানে সমস্ত ব্যবহারকারী যে সংস্করণটি ব্যবহার করছেন), যখুন অন্যটি আপডেট করা হয়।